### ISSUE TITLE
Missing dependency "maath"

### ISSUE THREAD

**Issue by tobua** [pmndrs/drei#1221]
- `@react-three/drei` version: 9.51.17
- `node` version: any

### Problem description:

Just ran into this issue after updating, the relevant release was published 15 minutes ago. The newly added dependency "maath" isn't listed in dependencies, but only devDependencies, therefore not available when downloaded from npm.

### Relevant code:

```
✘ [ERROR] Could not resolve "maath"

    node_modules/@react-three/drei/web/ScrollControls.js:6:23:
      6 │ import { easing } from 'maath';
```

### Suggested solution:

Move "maath" from "devDependencies" to "dependencies", that's all 😄

**Comment by iampava** [comment]
Can confirm I got the same bug.

**Comment by tobua** [comment]
Issue has been fixed and released by now with this commit aff17977804412faf87b6b9259f65c49a622e919, closing this.
